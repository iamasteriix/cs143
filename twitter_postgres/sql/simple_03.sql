SELECT count(*) FROM tweet_tags;

