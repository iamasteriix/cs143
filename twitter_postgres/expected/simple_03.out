 count 
-------
  2143
(1 row)

